src/plugins/orangehrmDashboardPlugin/Controller/DashboardController.php

```diff
@@ -22,18 +22,39 @@
 use OrangeHRM\Core\Controller\AbstractVueController;
 use OrangeHRM\Core\Vue\Component;
 use OrangeHRM\Framework\Http\Request;
+use OrangeHRM\Core\Service\ModuleService;
 use OrangeHRM\Core\Traits\Auth\AuthUserTrait;
 use OrangeHRM\Core\Helper\VueControllerHelper;
+use OrangeHRM\Core\Traits\ServiceContainerTrait;
+use OrangeHRM\Framework\Services;
 
 class DashboardController extends AbstractVueController
 {
     use AuthUserTrait;
+    use ServiceContainerTrait;
+
+    /**
+     * @return ModuleService
+     */
+    public function getModuleService(): ModuleService
```

Comment:
@RajithaKumara I've added some BE logic from reference. please check

Reply:
Will change getter method of following classes also;
```
src/plugins/orangehrmAdminPlugin/Api/ModulesAPI.php
src/plugins/orangehrmCorePlugin/Subscriber/ModuleNotAvailableSubscriber.php
```